docs/documenters/buildthedocs.rst

```diff
@@ -0,0 +1,20 @@
+Building the Docs Locally
```

Comment:
I'm not sure that I would consider "documenters" to be a separate role from developers at the moment. In future I can imagine that we will have _Translators_ as a distinct role.